GDJS/Runtime/runtimeobject.ts

```diff
@@ -232,8 +278,10 @@ namespace gdjs {
       oldObjectData: ObjectData,
       newObjectData: ObjectData
     ): boolean {
-      // If not redefined, mark by default the hot-reload as failed.
-      return false;
+      return this._hotReloadRuntimeObjectEffects(
```

Comment:
It should not be the responsibility of `updateFromObjectData`, I think, to update the effects. The hot reload can do it, as it has all it needs (old/new effect data, and access to remove/addEffect)

Reply:
Actually I'm confused why you would even need this knowing that this is implemented already in the hot reloader. Remove these changes entirely.

Reply:
I think this is dead code, I forgot to remove this (my mistake). I started with this before looking at the hot reloader, then I understood it should be a part of the hot reloader itself. I'll remove it in the next commit.